### ISSUE TITLE
Hard coded reaction rate and electrode charge transfer coefficients

### ISSUE THREAD

**Issue by noelhallemans** [pybamm-team/pybamm-eis#22]
The reaction rate and electrode charge transfer coefficient parameters are hard coded in the exchange current density function (through the Butler-Volmer kinetics). I believe these are parameters that vary from battery to battery. 

Wouldn't it make more sense to also consider these parameters in the parameter set? This would help for model parametrisation from impedance data as the reaction rate parameter could then represent the charge transfer resistance in a model where parameters are grouped.

**Comment by rtimms** [comment]
You can define them as `Parameter` objects inside the function, see [this example](https://github.com/pybamm-team/pybamm-eis/blob/main/examples/parameter_sweep.py)

**Comment by vsree345** [comment]
What if we can create a config file and load that dynamically, so that we would not have to change the source code every time?
this is how the config.json would look like
{
  "reaction_rate": 1.5e-5,
  "charge_transfer_coefficient": 0.5
}

**Comment by rtimms** [comment]
Yes, you could do that by following the example inked above to set up the `InputParameters` and then handling in reading in the config and setting the values.
